documentation/library_guide/overview.rst

```diff
@@ -61,9 +61,28 @@ does (see the |dpcpp_short| specification and the SYCL specification for details
 Known Limitations
 =================
 
-For ``transform_exclusive_scan``, ``transform_inclusive_scan`` algorithms, the result of the unary operation should be
-convertible to the type of the initial value if (one is provided), otherwise it is convertible to the type of values
-in the processed data sequence: (``std::iterator_traits<IteratorType>::value_type``).
+* For ``transform_exclusive_scan``, ``transform_inclusive_scan`` algorithms, the result of the unary operation should be
+  convertible to the type of the initial value if (one is provided), otherwise it is convertible to the type of values
```

Comment:
Looks like we do not need the parentheses here
```suggestion
  convertible to the type of the initial value if one is provided, otherwise it is convertible to the type of values
```